lib/Controller/PageController.php

```diff
@@ -166,7 +166,9 @@ public function index(): TemplateResponse {
 			Util::addTranslations('recognize');
 		}
 
-		$response = new TemplateResponse(Application::APP_ID, 'main');
+		$response = new TemplateResponse(Application::APP_ID, 'main', [
+			'pageTitle' => 'Photos'
```

Comment:
Would suggest that we do not add variables that aren't used, thoughts @artonge?

Reply:
Actually it's used in the server templates https://github.com/nextcloud/server/pull/36640 so please ignore :)